modules/Link.js

```diff
@@ -28,15 +29,19 @@ class Link extends React.Component {
     activeClassName: '',
     style: {},
     activeStyle: {},
-    isActive: (location, to, props) => {
-      return pathIsActive(
-        to.pathname,
-        location.pathname,
-        props.activeOnlyWhenExact
-      ) && queryIsActive(
-        to.query,
-        location.query
-      )
+    isActive: (location, to, props, router) => {
+      if (router) {
```

Comment:
@timdorr  Do you just want to spread the given props of the Link onto an anchor when there is no router and return that? Also, should it warn that it is doing this? If so, how should we go about firing a warning, `console.warn` causes the build to fail